entrypoint.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash -e
 
 AWS_REGION="${AWS_REGION:-us-east-1}"
+if [ "${APP_ENV}" != "" ]; then
```

Comment:
it would be really silly if someone did this, but if you have `APP_ENV` and `SERVICE_ENV` set, `APP_ENV` is going to overwrite `SERVICE_ENV`.  Again, very unlikely.

Reply:
I doubt this is an issue, because at the very least, they should be set to the same value.